language_detector/main.py

```diff
@@ -1,8 +1,11 @@
 # -*- coding: utf-8 -*-
 
 """This is the entry point of the program."""
-
 def detect_language(text, languages):
     """Returns the detected language of given text."""
     # implement your solution here
-    pass
+    lang_sums = {}
+    for language in languages
+        lang_sums[sum([text.count(" %s " %x) for x in language['common_words']])] = language['name']
+    return lang_sums[max(lang_sums.keys())]
+    #return lang_sums
```

Comment:
very concise and clean! tried hard to find things to improve on but couldn't find anything. the string formatting threw me off for a bit only because I hadn't used it in so long, but I gotta say it was a great use of it for this problem. Never even crossed my mind